lib/liftoff/cli.rb

```diff
@@ -35,7 +35,7 @@ def global_options
           @options[:strict_prompts] = strict_prompts
         end
 
-        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage') do |list|
+        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage,bundler') do |list|
```

Comment:
I just turned this on, and it runs counter to existing style, but we should follow it (and make note of places that the style needs to be updated)